packaging/suse/velum.spec.in

```diff
@@ -68,8 +68,8 @@ __PATCHEXECS__
 
 %build
 
-install -d vendor/cache
```

Comment:
it seems that bundler has this path `vendor/cache` as a hard requirement, it cannot be changed. even the `--path` option doesn't influence that. :-(